README.md

```diff
@@ -830,15 +830,15 @@ gross(usd); // ok
 gross(eur); // Type '"EUR"' is not assignable to type '"USD"'.
 ```
 
-[⇧ back to top](#flows-utility-types)
+[⇧ back to top](#table-of-contents)
 
 ---
 
 ## Flow's Utility Types
 
 ### `$Keys<T>`
 
-get the union type of all the keys in an object type `T`  
```

Comment:
I would like to get back these trailing whitespaces, because of a nice line-break to put link on a separate lines.

Reply:
What do you think about using inline HTML `<br>` tag instead? I think it's a bit more readable and searchable. Also, if we go and do a find&replace of trailing whitespace (or add it to the linter/prettier/editorconfig), we don't have to worry about messing up the double trailing whitespace.

If you prefer the double trailing whitespace in markdown. I can revert it.

Reply:
Great idea, I love it! 👍

Reply:
I added back the line breaks with `<br>`.

Also added alias for `Writable<T>`. Forgot to do it in #27.

Reply:
Alias looks good, although it's not re-exported from index.ts for public access.

Reply:
Whoops, my bad. I re-exported it in latest commit now.